docs/developer/best-practices/stability.asciidoc

```diff
@@ -43,11 +43,30 @@ dependency list!
 [discrete]
 === Test coverage
 
-* Does the feature have sufficient unit test coverage? (does it handle
-storeinSessions?)
```

Comment:
@stacey-gammon I didn't know what this meant but I'd be happy to add it back once I understand it better.